javascript/javascript1/week1/ezNamey.js

```diff
@@ -0,0 +1,8 @@
+const firstWords = ["trusty", "energetic", "obedient", "helpful", "dynamic", "dedicated", "encouraging", "openminded", "good", "punctual"];
+const secondWords = ["company", "corporation", "trade union", "trustworthy","popular", "fascinating", "powerful", "cheapest", "expensive","amazing"];
+let randomNumber1 = Math.floor(Math.random() * 10 + 0);
+let randomNumber2 = Math.floor(Math.random() * 10 + 0);
+let startUpName =firstWords[randomNumber1] + " " + secondWords[randomNumber2];
```

Comment:
These should be `const`s as they don't change.

Reply:
You don't need the `+ 0` part. I know it's in the homework, but it doesn't do anything :)

Reply:
The `startupName` can also be interpolated with backticks.